### DISCUSSION TITLE
zero1 + overlap_comm : Is this correct?

### DISCUSSION THREAD

**Discussion by OutisKwak** [deepspeedai/DeepSpeed#5512] [Q&A]
Why was the code below removed starting from version v0.13.0? 
`
overlap_comm = False
contiguous_gradients = False
`
https://github.com/microsoft/DeepSpeed/blob/c00388a2ef933f243b28d89bafb1b329d72557ad/deepspeed/runtime/engine.py#L1486

Can this ensure correctness during the overlap phase?
https://github.com/microsoft/DeepSpeed/blob/c00388a2ef933f243b28d89bafb1b329d72557ad/deepspeed/runtime/zero/stage_1_and_2.py#L1317

**Comment by tjruwase** [comment]
@Suparjie, thanks for the questions. Please see https://github.com/microsoft/DeepSpeed/pull/4887
